ui/package-lock.json

```diff
@@ -63,7 +63,7 @@
         "path-parse": "^1.0.7",
         "pluralize": "^8.0.0",
         "postcss": "^8.4.21",
-        "prop-types": "*",
+        "prop-types": "latest",
```

Comment:
Why these changes @dhiraj0911

Reply:
It's an auto-generated, I'll revert it back to previous state